pkg/storage/couchdb/couchdbstore_test.go

```diff
@@ -513,6 +513,9 @@ func TestCouchDBStore_Query(t *testing.T) {
 
 			err = itr.Release()
 			require.NoError(t, err)
+
+			require.Contains(t, mockLoggerProvider.MockLogger.WarnLogContents,
```

Comment:
I intended to have this check there. I think I somehow missed this before.

Reply:
Oh, looks like I did have it there originally but it got removed here: https://github.com/trustbloc/edge-core/pull/57. Not sure how I missed that, since it says I reviewed that PR. @bstasyszyn, do you remember why you removed that check?

Reply:
I can't remember. Maybe the tests didn't pass for me at the time.